projects/nft-import.md

```diff
@@ -0,0 +1,89 @@
+## Project Name <!-- Add your project name here with format "Project Name"-->
+NFT import 
+
+## Category <!--developer tooling, application, wallet, infrastructure, etc-->
+Tooling, WordPress plugin 
+
+## Project Description <!--Describe your project in a few sentences. -->
+NFT import is a WordPress Plugin allowing to import your NFTs in your WordPress media library, to include them in your website
+
+## Use of IPFS, Filecoin and Libp2p <!-- Describe how your project uses any or all of these technologies, and why. -->
+Import your NFTs images already stored in IPFS.
+
+Complementary to existing Kredeum NFTs WordPress plugin, that makes the other way: archive to IPFS and Mint NFTs.
+
+Achieving "sort of" import/export functionnality between WordPress and IPFS/NFTs
+
+## Project status <!--brainstorming, fundraising, under development, beta, shipped, etc-->
+Project started during ETHGlobal Web3WeekEnd Hackathon.
+
+Done: subgraph retreiving NFTs with **IPFS metadata** (extracting CID from tokenURI) from whatever eth network (mumbai/matic done).
+
+That allows to browse very easily any NFTs you own, on whatever blockchain... after deploying the subgraph to the network
+
+To be done : all the WordPress Plugin side, store the IPFS files retreived with the subgraph to the WordPress media Library
+
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+
+Subgraph preview : 
+https://thegraph.com/explorer/subgraph/zapaz/eip721-mumbai
+
+With this query :  
+{ tokens(first:9 where:{metadata_not:""}) { id tokenURI name description image metadata } } 
+
+![Subgraph preview](https://storageapi.fleek.co/zapaz-team-bucket/web3weekend/capture.png)
+
+WebPage listing latest mumbai NFTs using this subgraph : 
+https://storageapi.fleek.co/zapaz-team-bucket/web3weekend/index.html 
+
+## Target Audience <!--Describe who will be your project's users-->
+WordPress editors : photographs, artists, e-commerce, journalists, blogger...
+
+## Rough estimated user base (if applicable)
+WordPress is 40% of all web2 sites worldwide, possible target of hundred thousands editors 
+<!--How many users do you have right now?-->
+
+## Github repo <!--Attach a link to your GitHub repo - open source is required - please make sure your repo has a license file and is licensed using MIT open source license! -->
+https://github.com/Kredeum/ethglobal-web3
+
```

Comment:
The grants program requires MIT license, if you are applying for that.

Reply:
ok, not so used to licence differences

repo set to MIT licence now

Reply:
👍🏼🙏🏼